Resources/Prototypes/_Goobstation/Research/civilianservices.yml

```
@@ -0,0 +1,17 @@
+  id: MedicalDefense
+  name: research-technology-medical-defense
+  icon:
+    sprite: _Goobstation/Objects/Weapons/Guns/Cannons/syringe_gun.rsi
+    state: rapid-syringe-gun
+  discipline: CivilianServices
+  tier: 2
+  cost: 10000
+  recipeUnlocks:
+  - RapidSyringeGun
+
+# Tier 3
```

yamllint (1.35.1)

[warning] comment not indented like content

(comments-indentation)